packages/cli/src/index.ts

```diff
@@ -93,11 +93,13 @@ async function execCmd(args: string[]) {
   await Promise.all(
     packages.map((pkg) => {
       return execLimit(async () => {
-        let { code } = await spawn(args[0], args.slice(1), {
-          cwd: pkg.dir,
-          stdio: "inherit",
+        const { exitCode } = await exec(args[0], args.slice(1), {
+          nodeOptions: {
+            cwd: pkg.dir,
+            stdio: "inherit",
+          },
         });
-        highestExitCode = Math.max(code, highestExitCode);
+        highestExitCode = Math.max(exitCode ?? 1, highestExitCode);
```

Comment:
why do we need to treat a nullish `exitCode` as a failure here?

Reply:
Per `node:child_proccess.spawn()`, `exitCode` can be nullish in 2 cases:
 - The process failed to spawn.
 - The process was externally killed.
 
For our purpose, both of these mean that the underlying process never executed successfully, so we should tread a missing exit code as an error.